slides/slides.md

```diff
@@ -80,3 +80,32 @@ class: dense
 # Start here
 
 https://reactnative.dev/docs/environment-setup
+
+---
+class: dense
+---
+
+# Who's using it? - Community
+
+* Meta: Facebook, Oculus, Messenger Desktop
+* Microsoft: Office, Outlook, Teams, Xbox Game Pass, Skyp
+* Shopify
+* Wix
+* Pinterest
+* Tesla
+* Discord
+* Puma
+* Many more...
+
+---
+class: dense
+---
+
+# Who's using it? - NearForm Partners
```

Comment:
Not sure this is the best title - this reads as people we've worked with who are using it, not necessarily projects we have worked on.